src/rgw/rgw_admin.cc

```diff
@@ -8954,6 +8954,10 @@ int main(int argc, const char **argv)
       cerr << "ERROR: bucket not specified" << std::endl;
       return EINVAL;
     }
+    if (!gen.has_value()) {
```

Comment:
how would the user know which gen to specify?

Reply:
they don't, really. this is a hack on top of something that was kinda broken already